### DISCUSSION TITLE
Saving new note without any confirmation or popup

### DISCUSSION THREAD

**Discussion by avramukk** [coddingtonbear/obsidian-web#184] [Ideas]
First of all - the extension is amazing.
In most cases for me - it's just selecting text and press the shortcut ctrl+s.
Can you add some checkbox in settings like "skip confirmation popup"?

**Comment by coddingtonbear** [comment]
I'm afraid that every configuration option makes testing and verifying changes a lot more complicated; so I'm not up for adding an additional configuration option for that -- especially since it would prevent you from ever selecting which template you want to use.

That said, the dialog *is* keyboard accessible, so if you've set the shortcut to `<Ctrl+s>`, all you would need to do to submit immediately is press two keys: `<tab>` followed by `<enter>`.
